src/rouge/worker/worker.py

```
@@ -157,26 +247,70 @@ def execute_workflow(self, issue_id: int, description: str) -> bool:
+        Returns:
+            True if workflow executed successfully, False otherwise
+        """
+        try:
+            if status == "patch pending":
+                _, success = self._execute_patch_workflow(issue_id, description)
+            else:
+                _, success = self._execute_main_workflow(issue_id, description)
+            return success
 
         except subprocess.TimeoutExpired:
             self.logger.error(f"Workflow timed out for issue {issue_id}")
```

Ruff (0.14.11)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)